solutions/kubernetes/functions.sh

```diff
@@ -142,6 +142,20 @@ function scp_primary_node() {
         done
 }
 
+function scp_ceph_nodes() {
```

Comment:
This is only ceph related function so can we keep it in ceph codebase only?

Reply:
We can, I put there because for future functionality it may come in handy.